requirements.txt

```diff
@@ -4,17 +4,17 @@ click-plugins==1.1.1
 cligj==0.5.0
 coverage==5.3
 docutils==0.15.2
-Fiona==1.8.21
-geopandas==0.10.2
+Fiona==1.8.22
+geopandas==0.12.2
 harmony-service-lib==1.0.22
 munch==2.5.0
-netCDF4==1.5.5.1
-pandas==1.4.2
-numpy==1.22.3
-pyproj==3.1.0
+netCDF4==1.6.2
```

Comment:
A couple of these updates (like `netCDF4`, `rasterio` and `pyproj`) were needed to make things still work with GDAL==3.6.0.